### PR TITLE
Wire up on and off commands

### PR DESCRIPTION
**Pull request by LegitTalon**
@killtheliterate Does this look like the proper way to use your modules?

### CODE REVIEW

File: src/commands.js [hunk 1]
```diff
@@ -1,11 +1,54 @@
...
+
+import debug from 'debug'
+var error = debug('ephembot:commands')
+var log = debug('ephembot:commands')
+
+// 15 minutes is the default
+const defaultLevel = 15
+
+// talon - Denver-Devs:
+const token = 'xoxp-4015048983-4025456865-4309485153-f20f9c'
```
Comment (LegitTalon) [inline src/commands.js]: I just re-issued this token 'cause I'm not sure I should have committed it :see_no_evil:

Comment (killtheliterate) [inline src/commands.js]: I did the same thing, also thought about it last night like "hmm...well"

**Comment by killtheliterate** [comment]
The promise that get_history returns just needs to be tapped with a .then(), where you'll be able to pass the array to remove(). I may write a little map function to extract the values, but either way...

So, will `on` just give the setTimeout() a name so we can clear it in `off`?

```javascript
var prom = get(options)
prom.then(function(res) {
    remove('token', 'channel')(res)
})

**Comment by killtheliterate** [comment]
Looks like you have it wired up that way :8ball:

**Comment by LegitTalon** [comment]
Awesome. I think I'll add the `config` module. For the secret stuff and then bang out the dents by giving it a whirl

**Comment by LegitTalon** [comment]
Can grab some food and then test this out in #ephemeral

**Comment by killtheliterate** [comment]
Right on. I just hit on, but not sure if you deployed yet. No feedback from
the bot, which I suppose we can fix.

**Pull request merged**